File: 4.c
#define _CRT_SECURE_NO_WARNINGS	
#include <stdio.h>

int main(void)
{
	int number, j = 0;

	printf("Enter a number: ");
	scanf("%d", &number);
	
	for (int i = 0; i <= number; i++)
	{
		if (i != number)
		{
			printf("%d + ", i);
		}
		else if (i == number)
		{
			printf("%d = ", i);
		}
		j = j + i;
	}

	printf("%d\n", j);

	return 0;
}

File: 4.cs
// See https://aka.ms/new-console-template for more information
int j = 0;
Console.Write("Enter a number: ");
int number = Convert.ToInt32(Console.ReadLine());

for (int i = 0; i <= number; i++)
{
    if (i < number)
    {
        Console.Write(i + " + ");
    }
    else if(i == number)
    {
        Console.Write(i + " = ");
    }
    j = j + i;
}
Console.Write(j);
Console.WriteLine();
